initial privacy-policy page commit

8-MrAlex-8 opened the pull request.

8-MrAlex-8 committed:
initial privacy-policy page commit

gr2ws commented on app/src/main/java/lapuk_app/views/main/MainActivity.kt:
```diff
@@ -82,7 +83,9 @@ fun MainScreen() {
                         }
                         composable("info") {
-                            TODO()
+                            composable("privacy-policy") {
```
i think another item ni sa navcontroller di siya inside sa infocenter na composable, pero to define na under siya sa infocenter dapat composable("info/privacy-policy")

gr2ws commented on app/src/main/java/lapuk_app/views/main/ui/pages/PrivacyPolicy.kt:
```diff
@@ -0,0 +1,138 @@
+            topBar = {
+                TopBar()
+            },
+            content = { innerPadding ->
```
ang sulod ra sa content ang naa sa function, ang MainScreen na bahala sa navigation tas ang functions per page is only content

8-MrAlex-8 replied:
implemented

gr2ws commented on app/src/main/java/lapuk_app/views/main/ui/pages/PrivacyPolicy.kt:
```diff
@@ -0,0 +1,138 @@
+                            modifier = Modifier
+                                .padding(12.dp),
+                            text = "PRIVACY POLICY",
+                            fontSize = 36.sp
```
no need to individually declare font sizes, try style = Typography.titleLarge or something, basta naa nay bodyLarge bodyMedium tas bodySmall pero murag wala pay titles, pwede ra i isahan define sa Type.kt

gr2ws reviewed (commented):
shalan lagi ning review review boss akong gitry ra, pakilantaw ra sa latest commit sa segregate para naay slight na reference for pages

gr2ws commented on app/src/main/java/lapuk_app/views/main/ui/pages/PrivacyPolicy.kt:
```diff
@@ -0,0 +1,138 @@
+                        )
+                    }
+                    // navigation
+                    Box(modifier = Modifier.padding(vertical=18.dp), contentAlignment = Alignment.Center) {
```
consider sad diay making this scrollable na lang and di na need mag tap para next page, to simplify sad ang implementation

8-MrAlex-8 replied:
Di ra dagay nato ipugos sa usa ka container ssob kay gamay ra pud ang content

gr2ws replied:
sige sige butangi rag buttons para dili empty kaayo

gr2ws reviewed (commented):
karon ra nako na notice

8-MrAlex-8 committed:
di ko ka-freeze sa padding lol
anw, all changes as advised have been implemented.
scroll components are working well lang.

8-MrAlex-8 committed:
cleaning up warnings

gr2ws committed:
Merge branch 'main' into privacy-policy-branch
